role/permissions.py

```diff
@@ -0,0 +1,62 @@
+"""
+Definition of custom permissions
+"""
+
+from __future__ import unicode_literals
+
+
+class GroupTypes(object):  # pylint: disable=too-few-public-methods
+    """
+    Definition of generic group names
+    """
+    repo_administrator = '{}_repo_administrators'
+    repo_curator = '{}_repo_curators'
+    repo_author = '{}_repo_authors'
+
+
+class RepoPermission(object):
+    """
+    Permissions for the repo objects
+    Django permissions are defined as a tuple of (name, description)
+    """
+    view_repo = ('view_repo', 'Permission to view the repository')
```

Comment:
localization is a requirement for work on edX, but it's not a high priority for us. We can open an issue and address it later.